exonum/src/blockchain/schema.rs

```diff
@@ -464,4 +488,11 @@ impl<'a> Schema<&'a mut Fork> {
         self.configs_actual_from_mut().push(cfg_ref);
         // TODO: clear storages
     }
+
+    /// Adds transaction into persistent pool
+    #[doc(hidden)]
+    pub fn add_transaction_into_pool(&mut self, tx: RawMessage) {
```

Comment:
Now, this method as opposed to `transactions_mut` and `transactions_pool_mut` can arguably be made public, although, of course, it is still arguably too low-level for most applications.